src/main/java/com/jcabi/github/mock/MkStars.java

```diff
@@ -31,42 +31,101 @@
 
 import com.jcabi.aspects.Immutable;
 import com.jcabi.aspects.Loggable;
+import com.jcabi.github.Coordinates;
 import com.jcabi.github.Repo;
 import com.jcabi.github.Stars;
+import java.io.IOException;
+import java.util.List;
+import javax.validation.constraints.NotNull;
 import lombok.ToString;
-import org.apache.commons.lang3.NotImplementedException;
+import org.apache.commons.lang3.StringUtils;
+import org.xembly.Directives;
 
 /**
  * Github starring API.
- *
  * @author Paul Polishchuk (ppol@ua.fm)
  * @version $Id$
  * @since 0.15
- * @todo #919:30min Implement MkStars.starred() operation.
- * @todo #950:30min Implement MkStars.star() and MkStars.unstar() operations.
- *  Don't forget about unit tests.
  */
 @Immutable
 @Loggable(Loggable.DEBUG)
 @ToString
```

Comment:
Since you added the fields `storage`, `self`, and `coords`, we should make the class implement equal in terms of those. Let's add the annotation `@EqualsAndHashCode(of = { "storage", "self", "coords"})`.